src/main.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * @copyright 2018 Christoph Wurst <christoph@winzerhof-wurst.at>
```

Comment:
where does this file come from ?

adding a new big file like these sounds more than just "cleanup code" :smile:

Reply:
Just renaming from the old 'viewer.js' file